ui/app/src/services/configuration.ts

```diff
@@ -137,15 +137,6 @@ export function deserializeActiveTargetingModelData<T extends Object>(
   data: T,
   contentTypeFields: LookupTable<ContentTypeField>
 ): ContentInstance {
-  Object.keys(data).forEach((modelKey) => {
```

Comment:
This is not needed anymore as the service now returns the checkbox-group data as an array, not a comma-separated string